examples/workflow/cloudflare-workers/src/index.ts

```diff
@@ -0,0 +1,60 @@
+import { Client, Receiver } from "@upstash/qstash";
+import { serve } from "@upstash/qstash/workflow";
```

Comment:
we shouldn't use the base serve I think

I was thinking of adding a handler for hono and adding a cf example with hono router. I can handle it today